evopy/evopy.py

```diff
@@ -0,0 +1,53 @@
+"""Module used for the execution of the evolutionary algorithm."""
+import numpy as np
+
+from evopy.individual import Individual
+
+
+#pylint: disable=R0913
+# we have a lot of user facing arguments, so this warning should be ignored.
```

Comment:
Maybe it should be placed before this comment?

Reply:
I think this is more clear, you first see the ignored rule and then see the reason why.

Reply:
Fair point!